src/app/functions/server/getBreaches.ts

```diff
@@ -65,49 +65,21 @@ export async function getBreachIcons(breaches: Breach[]): Promise<LogoMap> {
     const breachDomains = breaches
       .map((breach) => breach.Domain)
       .filter((breachDomain) => breachDomain.length > 0);
-    const logoFolder = pathResolve(
-      dirname(fileURLToPath(import.meta.url)),
-      "../../../../public/logo_cache/"
-    );
-    try {
-      await mkdir(logoFolder);
-    } catch {
-      // Do nothing; if the directory already exists, that's fine.
-    }
-    const existingLogos = await readdir(logoFolder);
     // TODO: Batch to limit memory use?
     const logoMapElems = (await Promise.all(
-      breachDomains.map((breachDomain) => {
-        return new Promise((resolve, reject) => {
-          const logoFilename = breachDomain.toLowerCase() + ".ico";
-          const logoPath = pathResolve(logoFolder, logoFilename);
-          if (existingLogos.includes(logoFilename)) {
-            resolve([breachDomain, `/logo_cache/${logoFilename}`]);
-            return;
+      breachDomains.map(async (breachDomain) => {
+        const logoUrl = `https://s3.amazonaws.com/${
```

Comment:
Question: Should we add `https://s3.amazonaws.com/` to an `.env` variable as well or do we not expect this to be changed?

Reply:
In theory no, but I did expect us to use our own domain - I asked about that [here](https://mozilla.slack.com/archives/C04BLQVN151/p1687945004183639?thread_ts=1687809528.834059&cid=C04BLQVN151).

Reply:
Don't think I have access to that Slack channel/thread, but looking at the Wayback Machine, it looks like our previous Monitor CDN was something like https://monitor.cdn.mozilla.net/img/logos/Abandonia.png